### DISCUSSION TITLE
Add the function keys F13-F24 to the Bluetooth keyboard

### DISCUSSION THREAD

**Discussion by UUoocl** [AppGround-io/bluetooth-keyboard-and-mouse-support#268] [Ideas]
This idea was shared through email support, thank you for responding to my email. I wanted to post it here too. 

Could the Bluetooth Keyboard App be updated to include the F13 - F24 keys? 

I'm trying to set global Hotkeys for a program and I do not want to use hotkeys that may already be used by other programs.  The F13-F24 are not used in any program I use.  A custom layout made in the Bluetooth keyboard App with the F13-F24 keys would solve the problem.  

Thanks for this great app.  It's already very useful and your updates are making it even better!

**Comment by robin-collins** [comment]
Fantastic idea. Then could be paired with autohotkey or powertoys keyboard manager to create a full array of custom app keys
